01 Api Versioned/src/1.Api/Api/App_Start/SwaggerConfig.cs

```diff
@@ -0,0 +1,19 @@
+using System.Web.Http;
+using Swashbuckle.Application;
+
+namespace Api
+{
+    public class SwaggerConfig
+    {
+        public static void Configure(HttpConfiguration config)
+        {
+            config
+            .EnableSwagger(c =>
+            {
+                c.SingleApiVersion("v1", Core.Constants.ApiName);
```

Comment:
This value should be referenced with the api versioned, or maybe create access to different swagger config depending on the version